src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.de.xlf

```diff
@@ -495,6 +495,14 @@
                 <source>search-overlay.placeholder.page</source>
                 <target>Seiten durchsuchen...</target>
             </trans-unit>
+            <trans-unit id="75435af307a7ae8f965923968a0e0f94" resname="labels.error.content-save-desc">
+                <source>labels.error.content-save-desc</source>
+                <target>Seite konnte nicht gespeichert werden.</target>
+            </trans-unit>
+            <trans-unit id="75435af3a7a7a38f968923968a0e0f94" resname="labels.error.content-save-resource-locator">
+                <source>labels.error.content-save-resource-locator</source>
+                <target>Der gegebene Resource Locator ist nicht gültig.</target>
```

Comment:
for me it should be "Der angegebene ?Adresse? ist nicht gültig." /cc @berndhep what did you think?